Web Dev/Backend/Frameworks/JS based/Express JS.md

```diff
@@ -0,0 +1,114 @@
+
+## 🎓 Skills you can Master
+After completing this learning path, you will be knowledgable in:
+- Routing and middleware
+- Authentication and Authorization
+- Working with databases
+- Automated testing
+
+## 🛠️ Prerequisites
+
+- A solid understanding of [**JavaScript**](../../Languages/JavaScript.md) fundamentals is essential.
+
+- Knowledge of Node.js is essential since Express.js is built on it. You should know how to work with modules, create and run server-side JavaScript code, and interact with the file system.
+
+- Strong knowledge about HTTP methods, status codes, headers, and cookies since HTTP powers the web.
+
+- Basic understanding of the [**CLI**](../../../../Essentials.md) is important. This includes knowing how to navigate directories, create and edit files, and run commands.
+
+
+## 📲 Installation and Setup
+
+You need a code editor, browser and **Node.js** to follow along this learning path. You can choose the code editor and browser of your choice. The preferred ones are listed below:
+- **VS Code**
+
+    Light-weight code editor by Microsoft with a large ecosystem of plugins to help your workflow
+    
+    [Install VS Code ](https://code.visualstudio.com/download)
+- **Google Chrome**
+    
+    A browser based on V8 JavaScript engine with developer tools.
+    
+    [Install Google Chrome](https://www.google.com/chrome/)
+- **Node.js**
+
+    JavaScript runtime which helps you run JavaScript programs in your system.
+
+    [Install Node](https://nodejs.org/en/download/)
+
+    ***Note:*** Install the stable version
+    
+    This also installs npm(Node Package Manager) in the system which can be used for installing various dependencies, which are required for the development of the application, including Express.js
+    
+- **Express.js**
+
+    To install express.js on your device, first you should install node modules into your application. Then install express.js as a separate dependency. The following code can be implemented step-by-step in the terminal.
+    
+    ```
+  > npm install
+  > npm install express
+    ```
+    
+## 💡 Learning Session  
+
+**Fundamentals of Express JS**
+```
+> Understanding Node Architecture
+> Routing
+> Middleware
+> Views and templates
+> Static files
+> Node Package Manager (npm)
+```
+
+**🧑🏻‍💻 Learn from**
+
+Basically, Node.js and Express.js go hand-in-hand. So, all the tutorials for [**🎨 Node JS**](../../../FrontEnd/) can also be used for Express.js. Some other recommendations are:
+
+*Video Tutorials*
+- [Node.js Crash Course - NetNinja](https://www.youtube.com/playlist?list=PL4cUxeGkcC9jsz4LDYc6kv3ymONOKxwBU)
+- [Node.js and Express.js full course - FreeCodeCamp](https://www.youtube.com/watch?v=Oe421EPjeBE)
+
+
+*Articles/Blogs*
+
+- [Express.js: Fundamentals, by Edureka](https://medium.com/edureka/expressjs-tutorial-795ad6e65ab3)  
+
+- [Official Documentation](https://expressjs.com/)

```

Comment:
https://mannhowie.com/clean-architecture-node 

   this is a blog about clean architecture please add this here 👀